### ISSUE TITLE
detail about the paper

### ISSUE THREAD

**Issue by Daniellli** [ZexinHe/Neural-LightRig#2]
Hi, thank for such work for community. 
after going through your work. i  have some question:
1. does the 3X3 multi-light images pixel-align with input image?

**Comment by ZexinHe** [comment]
Hi, thank you for your interest in our work.

The ground-truth 3x3 multi-light images are pixel-aligned naturally, and we would love to see the generated results are well-pixel-aligned too. (They do in most of the cases)

However, as we do not implement any specific rule-based designs to the inference process, and the only contraint here is the diffusion training loss, small displacements do exist. We used grid-distortion augmentation in our paper, trying to close this gap.

Best
